preprocess/masks/make_moc_masks.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
This is a script for generating the MOC mask files.  Please let me know:
* If you're okay with where this is stored.  There would also be scripts in `preprocess/observations` for generating observations.
* If you're good with how the script works (e.g. just taking grid name and assuming the mask creator, mesh file, boundary extractor, etc. are linked locally).  Alternatively, one could supply the path to the MPAS-Tools and geometric_features repos, and the script would handle the details.  I'm not sure that's really easier...